website/docs/r/compute_instance.html.markdown

```diff
@@ -131,6 +131,11 @@ the type is "local-ssd", in which case scratch must be true).
 * `scratch` - (Optional) Whether the disk is a scratch disk as opposed to a
     persistent disk (required for local-ssd).
 
+* `interface` - (Optional) Specifies the disk interface to use for attaching this disk,
+    which is either SCSI or NVME. The default is SCSI. Persistent disks must always use
+    SCSI and the request will fail if you attempt to attach a persistent disk in any
```

Comment:
Would it be preferable to enforce the persistent disk restriction in the terraform code?